pkg/admission/certgenerator/certgenerator.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package certprovisioner
+package certgenerator
```

Comment:
Want to throw it out there from caller's code readability perspective:

We can probably have a package name `cert` under `admission`

caller's code will look like

```go
cg, err := cert.Generator() 
         
cw, err := cert.Writer()
```

Reply:
+1